engine/api/workflow_v2/gorp_model.go

```diff
@@ -14,8 +14,8 @@ type dbWorkflowRun struct {
 func (r dbWorkflowRun) Canonical() gorpmapper.CanonicalForms {
 	var _ = []interface{}{r.ID, r.ProjectKey, r.VCSServerID, r.RepositoryID, r.WorkflowName, r.WorkflowData, r.UserID, r.Contexts}
 	return gorpmapper.CanonicalForms{
-		"{{.ID}}{{.ProjectKey}}{{.VCSServerID}}{{.VCSServer}}{{.RepositoryID}}{{.Repository}}{{md5sum .WorkflowData}}{{.UserID}}{{md5sum .Contexts}}",
-		"{{.ID}}{{.ProjectKey}}{{.VCSServerID}}{{.VCSServer}}{{.RepositoryID}}{{.Repository}}{{hash .WorkflowData}}{{.UserID}}{{hash .Contexts}}",
+		"{{.ID}}{{.ProjectKey}}{{.VCSServerID}}{{.VCSServer}}{{.RepositoryID}}{{.Repository}}{{md5sum .WorkflowData}}{{.UserID}}",
+		// TODO add context
```

Comment:
why not adding a signature ?

Reply:
We have to migrate all existing resources that are not readable anymore